client/www/pages/dash/index.tsx

```diff
@@ -270,11 +271,8 @@ function Dashboard() {
     });
   }, [token]);
 
-  const apps = useMemo(() => {
```

Comment:
I read through the code here, and I think the useMemo in this case is not necessary. 

1. This just keeps the identity of the `apps` array stable, but imo nothing depends on the list identity being stable

Reply:
Is it not there to reduce the amount of re-rending that happens every time swr refetches?

Reply:
Afaik I don't think this will save us much. Here's my reasoning: 

1. This memo was saying: if _anything_ in  `data.apps` changes, then re-calculate `apps`
2. `swr` returns the states `data`, `error`, `isValidating`, `isLoading`. 
3. But if I understand correctly,  most of the time `data` is the thing that is going to be changing